### PR TITLE
Update kickstart repo sync test as per eval proposal

### PR DESCRIPTION
**Pull request by vsedmik**
### Problem Statement
`test_positive_sync_kickstart_repo` test case needs update as per eval proposal - see the Related Issue.

### Solution
This PR.

### Related Issues
https://issues.redhat.com/browse/SAT-25998

### PRT test Cases example
trigger: test-robottelo
pytest: tests/foreman/api/test_repositories.py -k sync_kickstart_repo

### CODE REVIEW

**Comment by vsedmik** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_repositories.py -k sync_kickstart_repo

**Review by vijaysawant — CHANGES_REQUESTED** [review]
Component name need to update.

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8249
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/api/test_repositories.py -k sync_kickstart_repo --external-logging
Test Result : ========== 10 deselected, 40 warnings, 3 errors in 1611.50s (0:26:51) ==========
```

File: tests/foreman/api/test_repositories.py [hunk 1]
```diff
@@ -144,31 +145,27 @@ def test_positive_epel_repositories_with_mirroring_policy(
...
-        5. Assert that sync was executed properly.
+        4. Assert that no errors related to encoding gzip are present in /var/log/messages.
+        5. Assert that sync was executed properly and kickstart content is present.
+
+    :expectedresults:
+        1. No encoding gzip errors present in /var/log/messages.
+        2. Sync succeeds and kickstart content is present.
 
-    :CaseComponent: Pulp
+    :customerscenario: true
```
[the remark this answers is not in the document]
Comment (vsedmik) [reply tests/foreman/api/test_repositories.py:165]: I'm inheriting the component from the module [here](https://github.com/SatelliteQE/robottelo/pull/16042/files#diff-18095fabdf2ee2e3af4b59f78c3af8ef49d7602590d3ba37f4f0369578d41673R7).

[the remark this answers is not in the document]
Comment (vijaysawant) [reply tests/foreman/api/test_repositories.py:165]: Okay got it, resolving comment

**Review by vijaysawant — APPROVED** [review]

**Review by sambible — APPROVED** [review]

**Pull request merged**